ivy_tests/test_ivy/helpers.py

```diff
@@ -1627,7 +1627,8 @@ def dtype_and_values(
     num_arrays=1,
     min_value=None,
     max_value=None,
-    safety_factor=0.95,
+    large_value_safety_factor=0.95,
```

Comment:
these should be `1.1` not `0.95`

Reply:
When I change it to 1.1, specifically for `uint64` and `int64`, it starts throwing an error: `Python int too large to convert to C long`. This is because of overflow and the number can no loger berepresented by `uint` or `int`. I have tried to handle this by including a check while generating `uint64` and `int64`